_posts/2016-09-21-approaching-tdd-outside-android.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+asset-type: post
+name: Approaching Outside-in TDD on Android I
+title: 'Approaching Outside-in TDD on Android I'
+author: Carlos Morera de la Chica
+image:
+    src: /assets/img/custom/blog/outside-in.png
+tags: 
+- android
+- tdd 
+- design
+canonical:
+    name: my personal blog
+    href: https://carlosmchica.github.io/approaching-tdd-outside-android/
+
+---
+
+<a href="http://martinfowler.com/articles/mocksArentStubs.html#DrivingTdd">Outside-in Test-Driven Development (TDD)</a> can be a challenge to implement. In this 3-part post series, we would like to share our experiences applying it to Android development and offer some practical tips for doing so yourself.
```

Comment:
@franziskas I was thinking about it but wasn't too sure how to fit it. I thought about adding it to the post introduction, but I'm not convinced the introduction is the best place to do it. Any ideas?